src/components/Home/Home.tsx

```diff
@@ -27,18 +41,67 @@ export function Home() {
       <br />
     </>
   ) : null;
+
+  const handleSubmit = () => {
+    const postalCodeRegex = /[a-zA-Z][0-9][a-zA-Z](-| |)[0-9][a-zA-Z][0-9]/;
+    if (postalCodeRegex.test(postalCode)) {
+      setRenderCards(true);
+      setShowPostalPrompt(false);
+    } else {
+      setShowInvalidPostal(true);
+    }
+  };
+  const afterPostalEntryMarkup = renderCards ? (
```

Comment:
`renderCards && (...)` i think would be more concise tho this totally works

Reply:
> `renderCards && (...)` i think would be more concise tho this totally works

Yes, you are correct, and the official docs even mention using the logical AND operator for conditional rendering, but there is a case to be made for using ternaries: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

For consistency's sake, I always use ternaries.